From 862ff4a328fff1b6c484082a3cdc209ee174faf5 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 28 Dec 2019 14:29:51 +0200
Subject: [PATCH 2/2] Improve Searchbar

  * from
  https://github.com/Havoc-OS/android_packages_apps_Launcher3/commit/8127ae42855bc7e922cd31c9f985dbb5a9c5fc33

	modified:   res/drawable/bg_all_apps_searchbox.xml
	modified:   res/layout/search_container_all_apps.xml
	modified:   res/values/dimens.xml

Change-Id: I62378a2feff3174e28a6ef3e5ae32a3eec7da967
---
 res/drawable/bg_all_apps_searchbox.xml   | 4 ++--
 res/layout/search_container_all_apps.xml | 4 +++-
 res/values/dimens.xml                    | 2 +-
 3 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/res/drawable/bg_all_apps_searchbox.xml b/res/drawable/bg_all_apps_searchbox.xml
index c3249279a..34465dd8b 100644
--- a/res/drawable/bg_all_apps_searchbox.xml
+++ b/res/drawable/bg_all_apps_searchbox.xml
@@ -15,5 +15,5 @@
 -->
 <shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
     <solid android:color="?attr/popupColorPrimary" />
-    <corners android:radius="2dp" />
-</shape>
\ No newline at end of file
+    <corners android:radius="24dp" />
+</shape>
diff --git a/res/layout/search_container_all_apps.xml b/res/layout/search_container_all_apps.xml
index fd9cb60fc..4717c3aa3 100644
--- a/res/layout/search_container_all_apps.xml
+++ b/res/layout/search_container_all_apps.xml
@@ -20,6 +20,8 @@
     android:layout_height="@dimen/all_apps_search_bar_field_height"
     android:layout_centerHorizontal="true"
     android:layout_gravity="top|center_horizontal"
+    android:layout_marginLeft="8dp"
+    android:layout_marginRight="8dp"
     android:background="@drawable/bg_all_apps_searchbox"
     android:elevation="1dp"
     android:focusableInTouchMode="true"
@@ -35,4 +37,4 @@
     android:textColor="?android:attr/textColorSecondary"
     android:textColorHint="@drawable/all_apps_search_hint"
     android:textSize="16sp"
-    android:translationY="24dp" />
\ No newline at end of file
+    android:translationY="24dp" />
diff --git a/res/values/dimens.xml b/res/values/dimens.xml
index d9b0f9a0b..921171817 100644
--- a/res/values/dimens.xml
+++ b/res/values/dimens.xml
@@ -79,7 +79,7 @@
     <dimen name="all_apps_background_canvas_height">475dp</dimen>
     <dimen name="all_apps_header_tab_height">50dp</dimen>
     <dimen name="all_apps_tabs_indicator_height">2dp</dimen>
-    <dimen name="all_apps_header_top_padding">36dp</dimen>
+    <dimen name="all_apps_header_top_padding">40dp</dimen>
     <dimen name="all_apps_work_profile_tab_footer_top_padding">16dp</dimen>
     <dimen name="all_apps_work_profile_tab_footer_bottom_padding">20dp</dimen>
     <dimen name="all_apps_tabs_side_padding">12dp</dimen>
-- 
2.17.1

